MANIFEST.in

```diff
@@ -16,17 +14,9 @@ exclude MANIFEST.in
 # Directories to exclude in PyPI package
 prune requirements
 prune tests
-prune docker
+prune *.egg-info
 prune typings
 prune scripts
 prune hooks
 prune docs
 prune .github
-
-# Patterns to exclude from any directory
-global-exclude *~
-global-exclude *.pyc
-global-exclude *.pyo
-global-exclude .ipynb_checkpoints
-global-exclude __pycache__
-global-exclude .DS_Store
```

Comment:
I made a new build with your PR and see lots of .DS_Store files included in the distribution

Reply:
i will add it back